wgengine/filter/filter_test.go

```diff
@@ -40,14 +40,27 @@ const (
 	testDeniedProto  ipproto.Proto = 127 // CRUDP, appropriately cruddy
 )
 
-func m(srcs []netip.Prefix, dsts []NetPortRange, protos ...ipproto.Proto) Match {
-	if protos == nil {
+func m(srcs []netip.Prefix, dsts []NetPortRange, opts ...any) Match {
```

Comment:
use of `any` deserves a comment

Reply:
oh its just a test, still worth a comment

Reply:
done